wlm_server/cache/channel.py

```diff
@@ -0,0 +1,65 @@
+"""Module for caching the channel status."""
+
+from collections import defaultdict
+
+from operation.models import Operation
+from setting.models import Setting
+
+class ChannelCache:
+    """Cache for channel status."""
+
+    def __init__(self):
+        # outer key: Channel__channel, inner key: User__username
+        self._channel_to_operation: defaultdict[int, dict[str, Operation]] = defaultdict(dict)
+        # key: Channel__channel
+        self._channel_to_setting: dict[int, Setting] = {}
+        self.load()
+
+    def load(self):
```

Comment:
No! I intended that `load()` would be called only once when the server starts.

Reply:
In that case, I think it should be protected (`_load()` for example).
If it is re-usable, I guess clearing should be better since admin want to roll-back the setting by deleting a record in the DB.

Reply:
Cool! I just made it private. For deleting a record, I will handle it later.